.circleci/base_config.yml

```
@@ -344,24 +344,35 @@ jobs:
           command: |
+            $CCacheFolderName = "D:/CircleCI-ccache"
+            if ( -not (Test-Path $CCacheFolderName)) {
+              New-Item $CCacheFolderName -ItemType Directory
+              Write-Host "$($CCacheFolderName) Created successfully"
+            }
             mkdir build
             # make sure we have the latest image!
-            docker pull arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-f3c77755ece
+            docker pull arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26
             # TODO - use preset
-            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-f3c77755ece cmake -G 'Visual Studio 17 2022' -T 'v143,host=x64' -DUSE_MAINTAINER_MODE=On -DUSE_GOOGLE_TESTS=On -DUSE_ENTERPRISE=On -DCMAKE_BUILD_TYPE=RelWithDebInfo -DPACKAGING=NSIS -DCMAKE_INSTALL_PREFIX=/ -DSKIP_PACKAGING=On -DUSE_FAILURE_TESTS=On -DSTATIC_EXECUTABLES=On -DOPENSSL_USE_STATIC_LIBS=On -DUSE_STRICT_OPENSSL_VERSION=On ..
+            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -v D:/CircleCI-ccache:C:/ccache -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26 cmake -G 'Visual Studio 17 2022' -T 'v143,host=x64' -DUSE_MAINTAINER_MODE=On -DUSE_GOOGLE_TESTS=On -DUSE_ENTERPRISE=On -DCMAKE_BUILD_TYPE=RelWithDebInfo -DPACKAGING=NSIS -DCMAKE_INSTALL_PREFIX=/ -DSKIP_PACKAGING=On -DUSE_FAILURE_TESTS=On -DSTATIC_EXECUTABLES=On -DOPENSSL_USE_STATIC_LIBS=On -DUSE_STRICT_OPENSSL_VERSION=On -DUSE_CCACHE=On ..
+      - run:
+          name: "Clear CCache statistics"
+          shell: powershell.exe
+          command: |
+            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -v D:/CircleCI-ccache:C:/ccache -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26 ccache -z
       - run:
           name: "Build"
           shell: powershell.exe
           command: |
-            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-f3c77755ece cmake --build . --config `"RelWithDebInfo`" --parallel -- /p:CL_MPcount=24
+            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -v D:/CircleCI-ccache:C:/ccache -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26 cmake --build . --config `"RelWithDebInfo`" --parallel -- /p:CL_MPcount=24 
+      - run:
+          name: "Print CCache statistics"
+          shell: powershell.exe
+          command: |
+            docker run --rm --isolation process -v D:/CircleCI/project:C:/project -v D:/CircleCI-ccache:C:/ccache -w C:/project/build arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26 ccache -s
```

We probably don't need the project directory, and we also don't care about the work dir
```suggestion
            docker run --rm --isolation process -v D:/CircleCI-ccache:C:/ccache arangodb/build-windows-x64:10.0.17763.4737-VS-17.3.3-OpenSSL-3.1.4-399cc6a6e26 ccache -s
```

---

Change in multiple places